packages/rpc/src/common.ts

```diff
@@ -39,7 +39,7 @@ type AssetSymbol = AssetAndChain['asset'];
 
 type UncheckedAssetAndChain = { asset: AssetSymbol; chain: Chain };
 
-type SwapFeeType = 'Network' | 'Ingress' | 'Egress';
+type SwapFeeType = 'IngressDepositChannel' | 'IngressVaultSwap' | 'Network' | 'Egress';
```

Comment:
the enum was updated in the backend in https://github.com/chainflip-io/chainflip-backend/pull/5573